packages/nimble-components/src/wafer-map/index.ts

```diff
@@ -58,6 +58,20 @@ export class WaferMap extends FoundationElement {
         colors: [],
         values: []
     };
+
+    @observable public renderReady = false;
+
+    public override connectedCallback(): void {
+        super.connectedCallback();
+        // The component has been attached to the DOM and the attributes, properties are available
+        // Use this for initialization/initial render
+
+        // Simulate the time required until render is complete, before displaying the wafermap
```

Comment:
Here also the "WaferMap" with "M" instead of "m"

Reply:
applied